concent_api/core/tests/test_integration_get_task_result.py

```diff
@@ -208,6 +208,7 @@ def test_concent_requests_task_result_from_provider_and_requestor_receives_failu
         with freeze_time("2017-12-01 11:00:12"):
             response_2 = self.client.post(
                 reverse('core:receive'),
+                data = '',
```

Comment:
Please remove this changes, they will come in different branch.

Reply:
Ok, I've deleted all [TMP] commits